src/pages/landing/ebooks/components/_hero.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import GetEbook from './_get-ebook'
+import { Flex } from 'components/containers'
+import { Header, QueryImage, Text } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+
+const MainWrapper = styled(Flex)`
+    width: 100%;
+    padding: 4% 3%;
+    background-image: ${(props) =>
+        props.bg1 ? props.bg1 : 'linear-gradient(281deg, #0e0e0e, #1b1b1b)'};
+
+    @media ${device.tablet} {
+        background-image: ${(props) =>
+            props.bgMobile ? props.bgMobile : 'linear-gradient(281deg, #1b1b1b, #0e0e0e)'};
+    }
+`
+const HeaderBody = styled(Flex)`
+    max-width: 1440px;
+    margin: 0;
+    @media ${device.tablet} {
+        flex-direction: column;
+        height: auto;
+    }
+`
+
+const TopHeaderImgWrapper = styled(Flex)`
+    justify-content: flex-start;
+    margin: 0;
+    padding: 0;
+
+    @media ${device.tablet} {
+        order: 2;
+    }
+`
+const HeaderImage = styled(QueryImage)`
+    margin: auto;
+    padding: 15px;
```

Comment:
I think it's better if we use `rem` instead of `px` for desktop screen size. If this is the case, please apply these changes to other components as well.